Failing test on Ubuntu 24.04 about PyFakeFS version (was: BIT 1.5.3 make test fail)

I am attempting to install backintime 1.5,3 (I have been using version 1.4.3 - that has become very slow).

Ubuntu 24.04 upgrade from 22.04

I had to force the install of pylint 3.3.0  with pip install --force pylint==3.3.0 --break-system-packages

I cloned the git file then ./configure&& make

Then did a make test

I am getting the error below

I am willing to test any patches.

Thanks
Stephen Carr

FAIL: test_001_depency_workaround (test.test_uniquenessset.General.test_001_depency_workaround)
Workaround until #1575 is fixed.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/sgcarr/Downloads/backintime/common/test/test_uniquenessset.py", line 65, in test_001_depency_workaround
    self.assertTrue(pyfakefs_version >= min_required_version)
AssertionError: False is not true

--- a/common/test/test_uniquenessset.py
+++ b/common/test/test_uniquenessset.py
@@ -62,7 +62,11 @@ class General(pyfakefs_ut.TestCase):
         pyfakefs_version = packaging.version.parse(pyfakefs.__version__)
         min_required_version = packaging.version.parse('5.7.0')
 
-        self.assertTrue(pyfakefs_version >= min_required_version)
+        self.assertTrue(
+            pyfakefs_version >= min_required_version,
+            'With Python 3.12 or later the PyFakeFS version should be '
+            f'minimum {min_required_version} '
+            f'or later but is {pyfakefs_version}.')
 
     def test_ctor_defaults(self):
         """Default values in constructor."""

